static/js/lib/auth.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+import "isomorphic-fetch"
+
+export const refreshAuthToken = () => {
```

Comment:
Can you write tests for these two functions?

Reply:
:woman_facepalming:

Reply:
I even had that in my checklist when I was working and just totally forgot, haha